cmd/admxgen/defs/mount.yaml

```diff
@@ -17,3 +17,25 @@
   type: "mount"
   meta:
     strategy: "append"
+
+- key: "/system-mounts"
+  displayname: "System mounts"
+  explaintext: |
+    Define network shares that will be mounted for the system.
+    If more shares are defined higher in other GPO, the entries listed here will be appended to the list and duplicates will be removed.
+    
+    Values should be in the format: <protocol>://<hostname-or-ip>/shared-dir.
+    Examples:
+      nfs://example.com/nfs_share
+      smb://example.com/smb_share
+      ftp://example.com/ftp_share
```

Comment:
ftp can be user@password ? As there is no kerberos tickets for those mounts, I think we should for all protocols, provides the user / password basic authentication scheme in the example.

Reply:
Well... This complicate things a lot. We don't account for this in the policy manager (nor anywhere else in the code, tbh). And, as those values will be stored in the system in a file (take the user mounts, for example), shouldn't we avoid having this kind of authentication for safety reasons?

Reply:
So, what’s the difference for system mount between:
`nfs://example.com/nfs_share` and `[anonymous]nfs://example.com/nfs_share`?

Reply:
The options that we define on the unit file, basically...

Reply:
Can you be a little bit more precise? (and how does it effect a mount where we don’t pass username/password?)

Reply:
Well, from what I've read, system mounts do not have a specific "anonymous" option for them. Which means I'll need to set some other options to specify that the user that is interacting with the mounted location shouldn't be checked and so on...